test/performance/README.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
+
+The purpose of this Kubernetes CNI scale testing tool is to evaluate the performance and scalability of
+CNI within Kubernetes clusters.
+
+The following sections detail these features and explain how to use the tool.
+
+## 1. Scalability
+
+Whether it's a cluster with a single node or hundreds of nodes, you can execute tests with just
+a command line.
+
+```shell
+make bin
+./bin/antrea-scale --kubeConfigPath=/root/.kube/config --timeout=120 --config=./test/performance/scale.yml
+```
+
+## 2. Configurability
+
+You can easily configure the test parameters to meet various testing scenarios through a YAML file.
+
+```yaml
+real_node: true
+repeat_times: 1
+namespace_num: 5
+pods_num_per_ns: 10
+svc_num_per_ns: 4
+np_num_per_ns: 5
+skip_deploy_workload: false
+teardown: false
+scales:
+  - name: "ScaleUpWorkloadPods"
+    package: "test/performance/framework"
+    repeat_times: 1
+  - name: "ScaleService"
+    package: "test/performance/framework"
+    repeat_times: 1
+  - name: "ScaleRestartAgent"
+    package: "test/performance/framework"
+    repeat_times: 1
+  - name: "RestartController"
+    package: "test/performance/framework"
+    repeat_times: 1
+  - name: "ScaleNetworkPolicy"
+    package: "test/performance/framework"
+    repeat_times: 1
+```
+
+## 3. Flexibility & Assemblability
```

Comment:
The following two parts are for configuration details, you can remove this title.